deployment/scripts/test-broker-failure.sh

```
@@ -0,0 +1,29 @@
+kubectl delete pod $KAFKA_POD -n retailstore
+
+echo "Broker deleted. Wait for operator to recreate..."
+
+# Wait for recreation
+kubectl wait --for=condition=ready pod -l strimzi.io/name=kafka-kafka -n retailstore --timeout=300s
+
+echo "Broker recovered successfully!"
+
+# Check consumer lag via Kafka command
+echo "Checking consumer lag for 'orders' topic..."
+kubectl exec -n retailstore -c kafka $KAFKA_POD -- bin/kafka-consumer-groups.sh --bootstrap-server localhost:9092 --describe --group order-group || echo "Cannot fetch lag yet"
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)